packages/editor/src/components/post-actions/actions.js

```diff
@@ -403,11 +401,71 @@ export const postRevisionsAction = {
 			post?._links?.[ 'version-history' ]?.[ 0 ]?.count ?? 0;
 		return lastRevisionId && revisionsCount > 1;
 	},
-	callback( posts ) {
+	callback( posts, onActionPerformed ) {
 		const post = posts[ 0 ];
 		const href = addQueryArgs( 'revision.php', {
 			revision: post?._links?.[ 'predecessor-version' ]?.[ 0 ]?.id,
 		} );
 		document.location.href = href;
+		if ( onActionPerformed ) {
+			onActionPerformed( posts );
+		}
 	},
 };
+
+export function usePostActions( { onActionPerformed } ) {
```

Comment:
What was the reasoning behind passing the sole argument in an object? Since we're dealing with a new and private API, I'd be more confident and just have:

```js
function usePostActions( onActionPerformed ) {
```